.pfnci/generate.py

```diff
@@ -175,6 +175,7 @@ def generate_dockerfile(self) -> str:
             f'RUN pyenv install {py_spec} && \\',
             f'    pyenv global {py_spec} && \\',
             '    pip install -U setuptools pip wheel',
+            '    pip install mypy==1.4.1',
```

Comment:
Let's test typing things in GitHub Actions instead of FlexCI with GPU, which is expensive.

```suggestion
```